src/emailservice/email_server.py

```diff
@@ -29,11 +29,10 @@
 from grpc_health.v1 import health_pb2
 from grpc_health.v1 import health_pb2_grpc
 
-from opencensus.trace.exporters import stackdriver_exporter
-from opencensus.trace.exporters import print_exporter
-from opencensus.trace.ext.grpc import server_interceptor
+from opencensus.ext.stackdriver import trace_exporter as stackdriver_exporter
+from opencensus.ext.grpc import server_interceptor
 from opencensus.common.transports.async_ import AsyncTransport
-from opencensus.trace.samplers import always_on
+from opencensus.trace import samplers
```

Comment:
Can you give some context on why you switched from `always_on` to `samplers` ? thanks!  is it just an updated opencensus package changing module names?

Reply:
Sure thing, I took it from this https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/commit/857d467a944035f7cc761e3c9a587d3ea8885f8b#diff-8d7603c858572605b8f095758097fbe02cf93cfdae10533244362e7176f4b164. Same for `recommendationservice`.

Reply:
Thanks!